### ISSUE TITLE
Cannot install using all-in-one method

### ISSUE THREAD

**Issue by edernucci** [wazuh/wazuh#7688]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| latest | - | - | - | Debian 4.19.171-2 (2021-01-30) x86_64 GNU/Linux |

Following the steps described in https://documentation.wazuh.com/4.0/installation-guide/open-distro/all-in-one-deployment/unattended-installation.html the following command was executed:

```
root@wazuh:~# curl -so ~/all-in-one-installation.sh https://raw.githubusercontent.com/wazuh/wazuh-documentation/4.0/resources/open-distro/unattended-installation/all-in-one-installation.sh && bash ~/all-in-one-installation.sh
```

And the result was as follows:

```
Starting the installation...
Installing all necessary utilities for the installation...
Done
Adding the Wazuh repository...
(23) Failed writing body
Done
Installing the Wazuh manager...
Error: Wazuh installation failed
```

**Comment by JavierBejMen** [comment]
Hi, 

I need more detailed info in order to help you, can you run the installation script with debug output enabled?

`bash ~/all-in-one-installation.sh -e`

**Comment by edernucci** [comment]
Hi @JavierBejMen, I think I have found the root of this problem.

tl; dr `E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation`

```
...
E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation
(23) Failed writing body
deb https://packages.wazuh.com/4.x/apt/ stable main
Hit:1 http://deb.debian.org/debian stretch-backports InRelease
Hit:2 http://debian-archive.trafficmanager.net/debian buster InRelease
Hit:3 http://debian-archive.trafficmanager.net/debian-security buster/updates InRelease
Hit:4 http://debian-archive.trafficmanager.net/debian buster-updates InRelease
Hit:5 http://debian-archive.trafficmanager.net/debian buster-backports InRelease
Get:6 https://packages.wazuh.com/4.x/apt stable InRelease [14.0 kB]
Err:6 https://packages.wazuh.com/4.x/apt stable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 96B3EE5F29111145
Reading package lists...
W: GPG error: https://packages.wazuh.com/4.x/apt stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 96B3EE5F29111145
E: The repository 'https://packages.wazuh.com/4.x/apt stable InRelease' is not signed.
```
Maybe adding gnupg to dependency list can solve this. Can you please add it to `all-in-one-installation.sh`?

**Comment by edernucci** [comment]
In time: apparently the `procps` package also needs to be added, as there is a dependency on the `free` command at the beginning of the process.

**Comment by JavierBejMen** [comment]
Were you able to install it?  I will check the dependencies issue with the department in charge.

**Comment by edernucci** [comment]
@JavierBejMen Yes, it worked. With the following command it was possible to have the functional installation:

```
apt update && \
apt install curl procps gpg && \
curl -so ~/all-in-one-installation.sh https://raw.githubusercontent.com/wazuh/wazuh-documentation/4.0/resources/open-distro/unattended-installation/all-in-one-installation.sh && bash ~/all-in-one-installation.sh
```

**Comment by JavierBejMen** [comment]
Thanks @edernucci for your feedback,

We will move forward and open a new issue to solve dependecies if needed.

Have a nice day!

**Comment by JavierBejMen** [comment]
Closes when #3546 is done.
